lib/components/util/sortOptions.ts

```diff
@@ -45,5 +61,8 @@ export const sortOptions = (
     }
   ]
 
-  return sortOptionsArray
+  return sortOptionsArray.filter((sortOption) =>
+    // @ts-expect-error TODO: why is this failing
```

Comment:
Create `type SortOptionEntry = { text: string, value: SortOptionsType }`, and declare `sortOptionsArray` to be of type `SortOptionEntry[]`.

Reply:
Great fix thanks!